templates/example_MPH_plots.ipynb

```diff
@@ -0,0 +1,185 @@
+{
```

Comment:
<p>Make sure to adjust the unit of the plot so it's actually in hours. </p><p>Also, pick like 20 or 30 FOVs where there's a clear decrease over time. There's some data on Creed that should show this, ask hadeesha </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>